.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+<!-- Write a description of what the PR solves and how -->
+
+<!-- Link to relevant Jira issue, add multiple if necessary -->
+Jira Issue: [RHELC-]()
+
+Checklist
+- [ ] PR meets acceptance criteria
+- [ ] PR has been tested manually in a VM (either author or reviewer)
+- [ ] Jira issue has been made public if possible
+- [ ] Code and tests are documented properly
+- [ ] The commits squashed to as few commits as possible (without losing data) <!-- The commits can be squashed to 1 commit, but then we might lose data regarding moving something to a new file and then refactoring for example. Hence squash without losing data -->
+- [ ] When merged: Jira issue has been updated to `Release Pending`
```

Comment:
I would mark this with something to identify that it is only for Convert2RHEL developers, external contributors don't need to care about it.

Reply:
I don't even think an external contributor would have permission to do so anyway. But it's more of a check for the one who merged it